yjit/src/stats.rs

```diff
@@ -198,6 +198,18 @@ make_counters! {
     send_args_splat_non_iseq,
     send_args_splat_cfunc,
     send_iseq_ruby2_keywords,
+    send_optimized_method_send_not_imm,
+    send_optimized_method_send_wrong_args,
+    send_optimized_method_send_null_mid,
+    send_optimized_method_send_null_cme,
+    send_optimized_method_send_nested,
+    send_optimized_method_send_chain,
+    send_optimized_method_send_chain_string,
+    send_optimized_method_send_chain_not_string,
+    send_optimized_method_send_chain_not_sym,
+    send_optimized_method_send_chain_not_string_or_sym,
+    send_optimized_method_send_getter,
```

Comment:
Nitpick but just for the sake of brevity could you rename these to `send_send_x` ?